modules/manage/partials/tiered-storage.adoc

```diff
@@ -1108,62 +1108,13 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true
-
-==== Caching and chunking properties
-
-Use the following cluster-level properties to set the maximum cache size, the cache check interval, and chunking qualities.
-ifndef::env-kubernetes[Edit them with the `rpk cluster config edit` command.]
-
-|===
-| Property      | Description
-
-| config_ref:cloud_storage_cache_check_interval,true,properties/object-storage-properties[]
-| The time, in milliseconds, between cache checks. The size of the cache can grow quickly, so it's important to have a small interval between checks. However, if the checks are too frequent, they consume a lot of resources. Default is 30000 ms (30 sec).
-
-| config_ref:cloud_storage_cache_chunk_size,true,properties/object-storage-properties[]
-| The size of a chunk downloaded into object storage cache. Default is 16 MiB.
-
-| config_ref:cloud_storage_cache_size_percent,true,properties/object-storage-properties[]
-| Maximum size (as a percentage) of the disk cache used by Tiered Storage. +
-If both this property and `cloud_storage_cache_size` are set, Redpanda uses the minimum of the two. 
-
-| config_ref:cloud_storage_cache_size,true,properties/object-storage-properties[]
-| Maximum size (in bytes) of the disk cache used by Tiered Storage. +
-If both this property and `cloud_storage_cache_size_percent` are set, Redpanda uses the minimum of the two.
-
-| config_ref:cloud_storage_chunk_eviction_strategy,true,properties/object-storage-properties[]
-| Strategy for evicting unused cache chunks, either `eager` (default), `capped`, or `predictive`.
-
-| config_ref:cloud_storage_disable_chunk_reads,true,properties/object-storage-properties[]
-| Flag to turn off chunk-based reads and enable full-segment downloads.
-
-| config_ref:cloud_storage_hydrated_chunks_per_segment_ratio,true,properties/object-storage-properties[]
-| The ratio of hydrated to non-hydrated chunks for each segment, where a current ratio above this value results in unused chunks being evicted. Default is 0.7.
-
-| config_ref:cloud_storage_min_chunks_per_segment_threshold,true,properties/object-storage-properties[]
-| The threshold below which all chunks of a segment can be hydrated without eviction. If the number of chunks in a segment is below this threshold, the segment is small enough that all chunks in it can be hydrated at any given time. Default is 5.
-|===
+By default, the cache directory is created in Redpanda's data directory, but it can be placed anywhere in the system. For example, you might want to put the cache directory on a dedicated drive with cheaper storage. Use the config_ref:cloud_storage_cache_directory,true,node-properties[] property on each broker to specify a different location for the cache directory. You must specify the full path.
```

Comment:
please check links: this is going to broker properties but link is now in object storage properties